server.js

```diff
@@ -3,26 +3,43 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+import netflixData from './data/netflix-titles.json'
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/books"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const Director = mongoose.model('Director', {
+  name: String,
+})
+
+const Title = mongoose.model('Title', {
+  title: String,
+
+  director: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Director'
+  }
+})
+
+if (process.env.RESET_DATABASE) {
+  const seedDatabase = async () => {
+    await Director.deleteMany()
+
+    netflixData.forEach(item => {
+      const director = new director(item)
+       director.save();
+    })
+  }
+
+  seedDatabase()
+}
```

Comment:
Well done, nice-looking code for seeding the database! But if I was correct with the "Show" above (please correct me if I'm wrong, Maks/Matilda/Van/Whoever reads this), then the seeding would look something like:

```
if (process.env.RESET_DATABASE) {
  const seedDatabase = async () => {
    await Show.deleteMany()

    netflixData.forEach(item => {
      const newShow = new Show(item)
      newShow.save();
    })
  }
  seedDatabase()
}
```